runtime/doc/manpages/nvim.1.md

```diff
@@ -0,0 +1,261 @@
+<!---
+Should be generated using md2man
+https://github.com/sunaku/md2man
+-->
+
+# NVIM 1 "April 2015" "Neovim" "User Commands"
+
+## NAME
+
+nvim - Vim out of the box
+
+## SYNOPSIS
+
+`nvim` [*options*] [*file*]...  
+`nvim` [*options*] `-`  
+`nvim` [*options*] `-t` <*tag*>  
+`nvim` [*options*] `-t` [*errorfile*]
+
+## DESCRIPTION
+
+`nvim` is an extensible text editor based off of Vim.
+
+While running `nvim`, consult the on-line help system with the ":help"
+command.
+
+If no files are specified, `nvim` will start with an empty buffer.
+Otherwise, one out of the following four may be used to choose one or more
+files to be edited.
+
+*file* ...
+  File(s) to edit.
+  The first one will be the current file and read into the buffer, where the
+  cursor will be position on the first line.
+  To switch to the other files use the ":next" command.
+
+`-`
+  The text to edit is read from stdin.
+  Commands are read from stderr, which should be a terminal.
+
+`-t` <*tag*>
+  The file to edit and the initial cursor position depends on a
+  tag, a sort of goto label.
+  *tag* is looked up in the tags file, the associated file becomes the current
+  file and the associated command is executed.
+  If *tag* is a function name, the file containing that function is opened
+  with the cursor positioned at the start of the function.
+  See ":help tag-commands".
+
+`-q` <*errorfile*>
+  QuickFix mode.
+  *errorfile* is read and the first error is displayed.
+  If *errorfile* is omitted, the filename is obtained from the 'errorfile'
+  option (defaults to *errors.err*).
+  Further errors can be jumped to with the ":cn" command.
+  See ":help quickfix".
+
+## OPTIONS
+
+`--`
+  Arguments after this will be unconditionally interpreted as files.
+  This can be used to edit files starting with a dash.
+
+`+`[*num*]
+  For the first file, the cursor will be positioned on line *num*.
+  If *num* isn't provided, the cursor will be positioned on the last line.
+
+`+/`<*pat*>
+  For the first file, the cursor will be positioned on the first
+  occurrence of *pat*.
+  See ":help search-pattern" for the available search patterns.
```

Comment:
Try `:[num]` in NeoVim. It will result in you going to line number `[num]`. I cannot say where this behaviour is described, but entering command-line range *without* entering a command will result in a jump to the last line in the range.

Thus `+1` and `+/pat` are the same thing as `+cmd`, not something special. `+` *without* arguments *is* a special case: empty command does not normally act as `+$` if you e.g. do `:<CR>` or `:execute ''<CR>`.